### PR TITLE
Annotated parameter names for events

### PR DESCRIPTION
**Pull request by WarFollowsMe**
After [PR#6684](https://github.com/paritytech/substrate/pull/6684) has been approved and merged, the description of events should be corrected according to the accepted rule.

### CODE REVIEW

**Comment by gavofyork** [comment]
@WarFollowsMe needsvresolving (poll was removed)

**Comment by bkchr** [comment]
@WarFollowsMe could you update this once again?

**Comment by WarFollowsMe** [comment]
@bkchr @gavofyork sorry for late response. Update is done.

File: runtime/parachains/src/inclusion.rs [hunk 1]
```diff
@@ -153,12 +153,12 @@ decl_error! {
 }
 
 decl_event! {
-	pub enum Event<T> where <T as frame_system::Trait>::Hash {
-		/// A candidate was backed.
+	pub enum Event<T> where <T as system::Trait>::Hash {
```
Comment (bkchr) [inline runtime/parachains/src/inclusion.rs:156]:
```suggestion
	pub enum Event<T> where <T as frame_system::Trait>::Hash {
```

**Review by bkchr — APPROVED** [review]

**Comment by bkchr** [comment]
bot merge

**Pull request merged**